mbs-s0003-20230501/README.md

```diff
@@ -0,0 +1,140 @@
+# Map-based noise simulations based on `scan-s0003` time-domain simulations
+
+Tag: `mbs-s0003-20230501`
```

Comment:
could you tag this release `mbs-s0013-20230501`?

Reply:
Ok